scripts/inventory/cobbler.py

```diff
@@ -52,7 +52,10 @@
 import os
 import re
 from time import time
-import xmlrpclib
+try:  # Python 3
+    from xmlrpc.client import Server
```

Comment:
@dagwieers Can you please add a changelog entry for this change?

Reply:
https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to for the latest information on that

Reply:
I honestly do not understand why we cannot use the PR subject as the changelog and have some guidelines.
It would be so much easier and simply requires an update to the PR subject before merging.

Automatic changelog would be just part of the Git and GitHub workflow.

Reply:
Nice Idea.

Reply:
@dagwieers we had this discussion multiple times earlier this year (when talking about changelogs for collections): often PR titles and commit messages suck as changelogs, because they are often not useful for users. (There are of course exceptions :) )